src/base/Utils.js

```diff
@@ -181,6 +185,21 @@ export function isPlainObject(obj) {
     return true;
 }
 
+export function cleanObject(obj, {skipEmptyString = true, filterFunc = null} = {}) {
```

Comment:
Again, cool function, but not sure where you'd use it. I think something similar would be necessary in api requests, but even there I think it's situational.